### ISSUE TITLE
Ability to create CAF-precursor art files

### ISSUE THREAD

**Issue by cjbackhouse** [SBNSoftware/sbncode#220]
Once CAFMaker is split in 2 (https://github.com/SBNSoftware/sbncode/issues/219), it will be reasonable to create "CAF precursor" files (better name/metaphor required) which are art files containing nothing but StandardRecord objects. These are logically equivalent to CAFs, but stored in art format. One would simply run the "StandardRecordMaker" but not "CAFMaker" step, and use an `outputCommands` to drop essentially everything but the StandardRecords.

The motivation to do this is to allow re-creating the CAFs, particularly with new systematic weights. What we are looking for here, then, is a fcl that does the above, and additionally allows whatever genie-level objects are necessary for computation of the genie weights to survice the outputCommands. As I write this, I realize we will also need some mechanism to understand which genie objects are associated to which reconstructed slice. Perhaps this information is already fully-retrievable from the StandardRecord structure. Perhaps we will need to add something.

**Comment by wesketchum** [comment]
Can this be closed @cjbackhouse ?

**Comment by cjbackhouse** [comment]
No, this is part of a series of issues detailing a longer-term plan. None of this has happened. Same for https://github.com/SBNSoftware/sbncode/issues/219 and https://github.com/SBNSoftware/sbncode/issues/221
